app/api/users/index.js

```diff
@@ -87,6 +87,10 @@ router.get('/sub/:sub', restricted, async (req, res) => {
           .status(401)
           .json({ message: 'This account has been deactivated' });
       }
+      
+      user.average_rating = user.total_stars/user.total_reviews;
+      delete user.total_stars;
+      delete user.total_reviews;
```

Comment:
**question**: I'm wondering if we want to keep the `user.total_reviews` field. `total_stars` doesn't really mean anything but `total_reviews` could be useful. Thoughts?